receiver/awscontainerinsightreceiver/receiver.go

```diff
@@ -236,7 +237,10 @@ func (acir *awsContainerInsightReceiver) initNeuronScraper(ctx context.Context,
 		Logger:                acir.settings.Logger,
 	}
 
-	acir.podResourcesStore = stores.NewPodResourcesStore(acir.settings.Logger)
+	acir.podResourcesStore, err = stores.NewPodResourcesStore(acir.settings.Logger)
+	if err != nil {
+		return err
+	}
```

Comment:
Will this have the desired behavior if the customer isn't running on a Neuron instance type?  Or if they are but haven't set things up right?  I would think we would want to log the error and then not initialize the `neuronMonitorScraper`, but otherwise proceed with normal agent startup.

Reply:
the error will basically be caught here : https://github.com/sam6134/opentelemetry-collector-contrib/blob/db8b745740632e976c11015184da5dfe28fbb049/receiver/awscontainerinsightreceiver/receiver.go#L118-L121 and the scrapper wont get initialized remaining nil. Before scrapping we check if the scrapper is nil or not

Reply:
Nice OK, missed that.  Looks good.